src/lib/hooks/useEmail.ts

```diff
@@ -12,9 +12,10 @@ export function useEmail() {
 
   const checkEmailConfirmed = !!serviceConfig?.checkEmailConfirmed
 
+  // Date set for 11 AM PST on the 18th of November
   const isNewEnough =
     !!profile?.createdAt &&
-    Date.parse(profile.createdAt) >= Date.parse('2024-11-16T02:00:00.000Z')
+    Date.parse(profile.createdAt) >= Date.parse('2024-11-18T02:19:00.000Z')
```

Comment:
Sorry if I'm missing something really obvious, doesn't that date and time correspond to 6:19PM on 17 November? 🤔

https://www.timeanddate.com/worldclock/converter.html?iso=20241118T021900&p1=136&p2=137

Reply:
heh, oops. ty.